parse/table_definitions_base/aerodrome/CLFactory_event_PoolCreated.json

```diff
@@ -31,7 +31,7 @@
             "name": "PoolCreated",
             "type": "event"
         },
-        "contract_address": "0x5e7bb104d84c7cb9b682aac2f3d509f5f406809a",
+        "contract_address": "SELECT b.pool FROM ref('Voter_event_GaugeCreated') a JOIN ref('CLFactory_event_PoolCreated') b ON a.pool = b.pool WHERE a.pool IS NOT NULL",
```

Comment:
@marcinja : Do we need to drop and recreate the table to make sure we don't have duplicate data from the old definition?